core/server/worker/src/main/java/alluxio/worker/file/FileSystemWorker.java

```diff
@@ -1,137 +0,0 @@
-/*
- * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
- * (the "License"). You may not use this work except in compliance with the License, which is
- * available at www.apache.org/licenses/LICENSE-2.0
- *
- * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied, as more fully set forth in the License.
- *
- * See the NOTICE file distributed with this work for information regarding copyright ownership.
- */
-
-package alluxio.worker.file;
-
-import alluxio.AlluxioURI;
-import alluxio.exception.FileAlreadyExistsException;
-import alluxio.exception.FileDoesNotExistException;
-import alluxio.worker.Worker;
-import alluxio.worker.file.options.CompleteUfsFileOptions;
-import alluxio.worker.file.options.CreateUfsFileOptions;
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-
-/**
- * Interface representing a file system worker.
- */
-public interface FileSystemWorker extends Worker {
```

Comment:
do we want to remove this interface? I thought the convention is to move and javadoc public methods of `DefaultFileSystemWorker` in this interface.